scripts/build.mjs

```diff
@@ -0,0 +1,84 @@
+import esbuild from "esbuild";
+import { copyFile, readFile, writeFile, rm } from "fs/promises";
+import { glob } from "glob";
+
+const sharedOptions = {
+  sourcemap: true,
+  minify: false,
+  allowOverwrite: true,
+  packages: "external",
+};
+
+async function main() {
+  // Start with a clean slate
+  await rm("pkg", { recursive: true });
+  // Build the source code for a neutral platform as ESM
+  await esbuild.build({
+    entryPoints: await glob(["./src/*.ts", "./src/**/*.ts"]),
+    outdir: "pkg/dist-src",
+    bundle: false,
+    platform: "neutral",
+    format: "esm",
+    ...sharedOptions,
```

Comment:
We were not emitting sourcemaps in `dist-src/` before so I think we should preserve that.

```suggestion
    ...sharedOptions,
    sorucemaps: false
```

Reply:
I'll have to test that to make sure it doesn't negatively effect the other builds

Reply:
This makes it so that the other builds reference `dist-src/`

Reply:
Let's just make it output the same as Pika for now, and we can revisit these points later